crates/starknet-server/src/api/utils.rs

```diff
@@ -86,3 +88,88 @@ impl TryFrom<&starknet_core::transactions::Transaction> for TransactionWithType
         Ok(transaction_with_type)
     }
 }
+
+impl TryFrom<&starknet_core::transactions::StarknetTransaction> for TransactionReceiptWithStatus {
```

Comment:
Shall be moved to `models/transaction.rs` where definition of `TransactionReceiptWithStatus` is.